### DISCUSSION TITLE
Tworzenie wysyłek z zamówień Allegro

### DISCUSSION THREAD

**Discussion by lasoczczyk** [allegro/allegro-api#8257] [Suggestions - rejected]
Przy tworzeniu paczek przez API (/shipment-management/shipments/create-commands) musimy podać pełne dane nadawcy, odbiorcy, dane paczki, wagę, pobrania, rodzaj wysyłki itd..... Przecież to wszystko jest w zamówieniu Allegro... 

Czy nie powinna istnieć dodatkowa funkcjonalność która tworzy paczkę na podstawie ID zamówienia? Czy można to jakoś obejść czy muszę przepisywać wszystkie dane z zamówienia? A może coś przegapiłem w dokumentacji?

**Comment by AureliuszBrussy** [comment]
Dziękuję za sugestię. Faktycznie, nie udostępniamy endpointu/funkcjonalności, która pozwalałaby utworzyć przesyłkę na podstawie samego numeru zamówienia.

**Comment by AureliuszBrussy** [comment]
Na ten moment po weryfikacji sugestia jest odrzucona - oznaczyliśmy to także w wątku, w którym już wystąpiła wcześniej: https://github.com/allegro/allegro-api/discussions/7769.
